CONTRIBUTING.md

```diff
@@ -265,36 +171,97 @@ $ git lfs pull
 
 [git-lfs]: https://git-lfs.github.com
 
-## Issues
+Any contribution that touches the kernel (i.e., anything in `pkg/arvo/sys`),
+should be accompanied by an updated [solid pill](#the-kernel-and-pills).  Pills
+are tracked in the repository via [git LFS][git-lfs].
 
-The [GitHub tracker][issu] is our canonical source of truth around issues,
-bugs, performance problems, feature requests, and so on.  If you encounter any
-issues when developing on Urbit, feel free to submit a report about it here.
+Whenever you make a contribution to the kernel, please create a new solid pill
+via:
 
-A good bug report, description of a crash, etc., should ideally be
-*reproducible*, with clear steps as to how another developer can replicate and
-examine your problem.  That said, this isn't always possible -- some bugs
-depend on having created a complicated or unusual state, or can otherwise
-simply be difficult to trigger again (say, you encountered it in the last
-continuity era).
+```
+sh/update-solid-pill
+```
 
-Your issue should thus at a minimum be *informative*.  The best advice here is
-probably "don't write bad issues," where "bad" is a matter of judgment and
-taste.  Issues that the maintainers don't judge to be sufficiently useful or
-informative may be closed.
+You should include the updated pill in the same commit that updates the source.
 
-Feature requests are welcome, but they should include sufficient detail and
-explanation, as well as a discussion of perceived benefits one could expect
-from them.  "It would be cool if.." probably does not, in itself, constitute a
-good feature request; instead, try to be specific about what you're requesting,
-and what your desired feature would accomplish.
+Since anything under [`pkg/arvo/sys/`][sys]) is bootstrapped from a pill, it
+must be recompiled if any changes are made. This should happen automatically
+when you make changes, but if it doesn't, the command to manually recompile and
+install the new kernel is `|reset` in `dojo`.  This rebuilds from the `sys`
+directory in the `base` desk in `%clay`.
+
+Currently, `|reset` does not reload apps like `dojo` itself, which will still
+reference the old kernel. To force them to reload, make a trivial edit to their
+main source file (under the `app` directory) in `%clay`.
+
+[arvo]: https://github.com/urbit/urbit/tree/master/pkg/arvo
+[sys]: https://github.com/urbit/urbit/tree/master/pkg/arvo/sys
```

Comment:
This feels outdated. I haven't needed to be concerned about this in a long time.

Reply:
We _may_ want to mention the ability to build pills off non-base desks though, somewher in the section above.

If, during development, you don't want to bother writing state migration logic yet, you can `|new-desk %new-base`, mount that, put the new files in there, then `+solid %new-base` to build the pill off that desk.

Reply:
I'm going to say the latter is out of scope (this doc is already probably too in the weeds)